tests/util.py

```diff
@@ -82,3 +85,17 @@ def make(self, fname: Optional[str] = None, size: int = 2**10) -> Path:
         random_str = "".join(random.choices(string.ascii_letters + string.digits, k=size))
         random_file.write_text(random_str, encoding="utf-8")
         return random_file
+
+
+def commit_random_file_on_branch(
+    random_file_factory: RandomFileFactory, fs: LakeFSFileSystem, repository: str, temp_branch: str
+) -> None:
+    random_file = random_file_factory.make()
+    lpath = str(random_file)
+    rpath = f"{repository}/{temp_branch}/{random_file.name}"
+
+    fs.put(lpath, rpath, precheck=False)
+
+    commit = client_helpers.commit(
+        client=fs.client, repository=repository, branch=temp_branch, message="Commit File Factory"
```

Comment:
```suggestion
        client=fs.client, repository=repository, branch=temp_branch, message=f"Add file {random_file.name!r}"
```

See the other similar comment - can also be done in a follow-up if you prefer.